tst/EnergyPlus/unit/HVACVariableRefrigerantFlow.unit.cc

```diff
@@ -16149,4 +16149,4229 @@ TEST_F(EnergyPlusFixture, VRF_Condenser_Calc_EIRFPLR_Bound_Test)
+        "    23.88889,                !- Minimum Value of y",
+        "    46.11111,                !- Maximum Value of y",
+        "    ,                        !- Minimum Curve Output",
+        "    ,                        !- Maximum Curve Output",
+        "    Temperature,             !- Input Unit Type for X",
+        "    Temperature,             !- Input Unit Type for Y",
+        "    Dimensionless;           !- Output Unit Type",
+
+        "Curve:Biquadratic,",
+        "    VarSpeedCoolCapLSFT,     !- Name",
+        "    0.476428E+00,            !- Coefficient1 Constant",
+        "    0.401147E-01,            !- Coefficient2 x",
+        "    0.226411E-03,            !- Coefficient3 x**2",
+        "    -0.827136E-03,           !- Coefficient4 y",
+        "    -0.732240E-05,           !- Coefficient5 y**2",
+        "    -0.446278E-03,           !- Coefficient6 x*y",
+        "    12.77778,                !- Minimum Value of x",
+        "    23.88889,                !- Maximum Value of x",
+        "    23.88889,                !- Minimum Value of y",
+        "    46.11111,                !- Maximum Value of y",
+        "    ,                        !- Minimum Curve Output",
+        "    ,                        !- Maximum Curve Output",
+        "    Temperature,             !- Input Unit Type for X",
+        "    Temperature,             !- Input Unit Type for Y",
+        "    Dimensionless;           !- Output Unit Type",
+
+        "Curve:Biquadratic,",
+        "    VarSpeedCoolEIRLSFT,     !- Name",
+        "    0.774645E+00,            !- Coefficient1 Constant",
+        "    -0.343731E-01,           !- Coefficient2 x",
+        "    0.783173E-03,            !- Coefficient3 x**2",
+        "    0.146596E-01,            !- Coefficient4 y",
+        "    0.488851E-03,            !- Coefficient5 y**2",
+        "    -0.752036E-03,           !- Coefficient6 x*y",
+        "    12.77778,                !- Minimum Value of x",
+        "    23.88889,                !- Maximum Value of x",
+        "    23.88889,                !- Minimum Value of y",
+        "    46.11111,                !- Maximum Value of y",
+        "    ,                        !- Minimum Curve Output",
+        "    ,                        !- Maximum Curve Output",
+        "    Temperature,             !- Input Unit Type for X",
+        "    Temperature,             !- Input Unit Type for Y",
+        "    Dimensionless;           !- Output Unit Type",
+
+        " !-   ===========  ALL OBJECTS IN CLASS: FLUIDPROPERTIES:NAME ===========            ",
+        "                                                                                     ",
+        "   FluidProperties:Name,                                                             ",
+        "     R410a,                   !- Fluid Name                                          ",
+        "     Refrigerant;             !- Fluid Type                                          ",
```

Comment:
Ugh. Now we have these 1200+ lines of fluid properties repeated 5 times in this source file. This screams for a VRF fixture or at least a `constexpr string_view` that could be defined once at the top and appended to the various input strings. Not saying this has to be part of this PR, but soon, please.

Reply:
While you're at it, you may want to create a version of `process_idf` that takes a `gsl::span<std::string_view>` rather than a `std::string &`. I'm not sure how long it takes to concatenate 1000+ strings, but however long it is we shouldn't be doing it just to turn around and immediately parse the string again.

Reply:
Maybe should be handled in a separate issue.